docker/convert.go

```diff
@@ -29,6 +30,21 @@ func isVolume(s string) bool {
 	return !isBind(s)
 }
 
+func toAbs(s string) string {
```

Comment:
Even though this is a not exported, can you add a doc string so others who come across know the what this function is specifically for?